lib/Controller/PageController.php

```diff
@@ -92,10 +100,22 @@ public function index(): TemplateResponse {
 
 		$user = $this->userManager->get($this->uid);
 
+		$storageInfo = $this->helper::getStorageInfo('/');
+		if ($storageInfo['quota'] !== FileInfo::SPACE_UNLIMITED) {
+			$totalSpace = $this->l->t('Unlimited');
+		} else {
+			$totalSpace = $this->helper::humanFileSize($storageInfo['total']);
+		}
+
 		$this->initialState->provideInitialState(
 			'personalInfoParameters',
 			[
 				'languageMap' => $this->getLanguageMap($user),
+				'quota' => $storageInfo['quota'],
+				'totalSpace' => $totalSpace,
+				'freeSpace' => $this->helper::humanFileSize($storageInfo['free']),
+				'usage' => $this->helper::humanFileSize($storageInfo['used']),
+				'usageRelative' => round($storageInfo['relative']),
```

Comment:
suggestion(optional): depending on final usage of inital state variables, one of them could possibly be removed.

Reply:
✅